src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -94,7 +94,9 @@ public partial class CryptoConfig
         public static bool AllowOnlyFipsAlgorithms { get { throw null; } }
         public static void AddAlgorithm(System.Type algorithm, params string[] names) { }
         public static void AddOID(string oid, params string[] names) { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Types might be removed")]
```

Comment:
It can't be succinct, just due to the open nature of the method, so "The default algorithm implementations might be removed, use strong type references instead." is about the best I can do.

Things someone might want, depending on the value they passed in:

* MD5.Create()
* SHA1.Create()
* SHA256.Create()
* SHA384.Create()
* SHA512.Create()
* new HMACMD5(...)
* new HMACSHA1(...)
* new HMACSHA256(...)
* new HMACSHA384(...)
* new HMACSHA512(...)
* RSA.Create()
* DSA.Create()
* ECDsa.Create()
* Aes.Create()
* TripleDES.Create()
* DES.Create()
* RC2.Create()
* new X509BasicConstraintsExtension(...)
* new X509SubjectKeyIdentifierExtension(...)
* new X509KeyUsageExtension(...)
* new X509EnhancedKeyUsageExtension(...)
* new X509Chain(...)
* new Pkcs9ContentType(...)
* new Pkcs9MessageDigest(...)
* new Pkcs9SigningTime(...)
* new Pkcs9DocumentName(...)
* new Pkcs9DocumentDescription(...)
* Any fully qualified type name (since it defers to Type.GetType())
* Anything they registered themselves with CryptoConfig.AddAlgorithm

Reply:
> The default algorithm implementations might be removed, use strong type references instead.

That one should work. Maybe add an example or two so people have something to go off?